apps/homeless-people-help-map/src/components/App.tsx

```diff
@@ -71,7 +74,9 @@ export const App = () => {
   const mapRef = useRef<MapHandle>(null);
 
   const [selectedMarker, setSelectedMarker] = useState<Feature<Point> | null>(null);
+  // terrain services are located in sidebar
   const [selectedFeature, setSelectedFeature] = useState<Feature | null>(null);
+  // grouped terrain services are shown on map
```

Comment:
is selectedFeature just a sidebar thing ? then maybe a better name

comment detailin g why there is both selectedMarker and selectedFeatures likely needed as well

Reply:
ok, adding a TODO comment is fair I guess 😆